<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaAdicionar</class>
 <widget class="QWidget" name="JanelaAdicionar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="cxPeso">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>266</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="cxAnotacoes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendario">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>288</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Peso Atual</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>319</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Digite aqui qualquer anotacao</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Kg</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listaRefeicao">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>20</y>
     <width>271</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QListWidget" name="listaAlimentos">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>261</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Desktop/refeicao.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>140</y>
     <width>931</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Desktop/seta2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Desktop/fundo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btSalvar">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>490</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listaRefeicao</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>JanelaAdicionar</receiver>
   <slot>removerAlimento()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listaAlimentos</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>JanelaAdicionar</receiver>
   <slot>adicionarAlimento()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>671</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSalvar</sender>
   <signal>clicked()</signal>
   <receiver>JanelaAdicionar</receiver>
   <slot>salvar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>removerAlimento()</slot>
  <slot>adicionarAlimento()</slot>
  <slot>salvar()</slot>
 </slots>
</ui>
